test/compareLocations.test.js

```diff
@@ -0,0 +1,152 @@
+var should = require("should");
+var compareLocations = require("../lib/compareLocations");
+var createLocation = function(overides) {
```

Comment:
The function here is named `createLocation`, however, it actually create a `Position` object rather than a `SourceLocation`, according to [estree spec](https://github.com/estree/estree/blob/master/es5.md#node-objects) respected by acorn.
The tests pass because they follow the code logic, rather than the real purpose.